vscode/src/documentation.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getCompilerWorker } from "qsharp-lang";
+import { isQsharpDocument } from "./common";
+import { getTarget } from "./config";
+import { Uri, window } from "vscode";
+import { loadProject } from "./projectSystem";
+import { sendMessageToPanel } from "./webviewPanel";
+
+export async function showDocumentationCommand(extensionUri: Uri) {
+  // Reveal panel and show 'Loading...' for immediate feedback.
+  sendMessageToPanel(
+    "documentation", // This is needed to route the message to the proper panel
+    true, // Reveal panel
+    null, // With no message
+  );
+  const editor = window.activeTextEditor;
+  if (!editor || !isQsharpDocument(editor.document)) {
```

Comment:
You should probably do this check _before_ you open the panel in line 13 above.

Reply:
Yes! Updated.